### DISCUSSION TITLE
Esp32-s3 qualia

### DISCUSSION THREAD

**Discussion by JavierTolvia** [bitbank2/AnimatedGIF#95] [Q&A]
Hello,
I was trying to use a rount tft display with the adafruit gfx library. I was following the example to create a rainbow and everything worked fined (https://learn.adafruit.com/adafruit-qualia-esp32-s3-for-rgb666-displays/arduino-rainbow-demo) but then I tried to display a gif using this library as I had good results using it for another tft screen.
My problem is that the screen is defined with a different class than the one that it is used for the example with the adafruit gfx library, so the methods give me some errors.
I wanted to ask if someone could help me adapt the GifDraw function to be able to run everything on my screen with the esp32-s3 qualia.
Thanks in advance for any answer,
Javier

**Comment by bitbank2** [comment]
Hi Javier,
The "someone" is me, the author of this code. Unfortunately I don't have time to give free tech support to people needing help with their personal projects.
